.gitignore

```diff
@@ -3,4 +3,7 @@
 *.crt
 *.key
 *.srl
-*.csr
\ No newline at end of file
+*.csr
+oidc/data
```

Comment:
Not really a big deal but we can delete the oidc/data line now that the files are no longer persisted there (I should have done that earlier)